src/scala/com/github/johnynek/bazel_deps/Writer.scala

```diff
@@ -186,7 +186,23 @@ object Writer {
       val langFn = language(g, model)
       def replacedTarget(u: UnversionedCoordinate): Option[Target] =
         Label.replaced(u, model.getReplacements).map { case (lab, lang) =>
-          Target(lang, Label.localTarget(pathInRoot, u, lang), exports = Set(lab))
+          // TODO: converge on using java_import instead of java_library:
+          // https://github.com/johnynek/bazel-deps/issues/102
+          lang match {
+            case Language.Java =>
+              Target(lang,
+                kind = Target.Library,
+                name = Label.localTarget(pathInRoot, u, lang),
+                exports = Set(lab),
+                jars = Set.empty)
+            case _: Language.Scala =>
+              Target(lang,
+                kind = Target.Import,
+                name = Label.localTarget(pathInRoot, u, lang),
+                exports = Set.empty,
+                jars = Set(lab))
```

Comment:
here is the bug. I don't think we should be changing replacements actually.

@bluedragonx do you want to try reverting just this block here and see if that fixes for you?

Would be nice to add a test that replacements don't get handled with scala_import (and instead just use scala_library export).

Reply:
Ah yes. I didn't understand what the replacements are and went too fast over them. Now I see they're described in the README and indeed they should be handled by scala_library.